script/testing/oltpbench/run_oltpbench_v2.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import traceback
+import json
+import itertools
+
+base_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.insert(0, base_path)
+
+from oltpbench.test_case_oltp import TestCaseOLTP
+from oltpbench.test_oltpbench_v2 import TestOLTPBenchV2
+from oltpbench.utils import parse_command_line_args
+from oltpbench import constants
+
+def generate_test_suite(args):
+    args_configfile = args.get("config_file")
+
+    oltp_test_suite = []
+
+    if not args_configfile:
+        msg = "config file is not provided"
+        raise FileNotFoundError(msg)
+
+    configfile_path = os.path.join(os.getcwd(), args_configfile)
+    if not os.path.exists(configfile_path):
+        msg = "Unable to find OLTPBench config file '{}'".format(
+            args_configfile)
+        raise FileNotFoundError(msg)
+
+    oltp_test_suite_json = None
+    with open(configfile_path) as oltp_tracefile:
+        oltp_test_suite_json = json.load(oltp_tracefile)
+
+    if not oltp_test_suite_json:
+        msg = "Unable to load OLTPBench config file '{}'. Maybe it is not valid JSON file?".format(
+            args_configfile)
+        raise TypeError(msg)
+
+    # publish test results to the server
+    oltp_report_server = constants.PERFORMANCE_STORAGE_SERVICE_API[args.get("publish_results")]
```

Comment:
Can we add error handling in case they pass in a value that we don't support. Just a little `args.get('publish_results') in constants.PERFORMANCE_STORAGE_SERVICE_API.keys()`. then we can give them a nice error message if they pass in the wrong thing

Reply:
The error handling is implemented by argparse : https://docs.python.org/3/library/argparse.html#choices
We don't need to do extra work.